tests/e2e/support/environment/userManagement.ts

```diff
@@ -8,6 +8,7 @@ import {
   federatedUserStore
 } from '../store'
 import { config } from '../../config'
+import { dummyKeycloakGroupStore } from '../store/keycloak'
```

Comment:
why not?
```suggestion
  federatedUserStore,
  dummyKeycloakGroupStore
} from '../store'
import { config } from '../../config'
```

Reply:
very good question. I was confused because of existing code so i created issue for that https://github.com/owncloud/web/issues/11927

Reply:
This can be fixed in this PR. other will be done later on

Reply:
Sorry, but if I'm going to fix all of those past merged code in this PR then my PR takes longer time.

Reply:
You do not have to refactor all other code. just make your [new] code stick with https://github.com/owncloud/web/issues/11927 and we can do other remaining in coming PRs

Reply:
i think you didn't look latest commit throughly, i already did that 😄

Reply:
this file has no latest change. the review is not marked outdated